saleor/core/utils/random_data.py

```diff
@@ -1438,7 +1434,9 @@ def create_vouchers():
 
 
 def create_gift_cards(how_many=5):
-    product_pk = Product.objects.get(name="Gift card").pk
+    product_pk = Product.objects.get(name="Gift card 100").pk
```

Comment:
`get` will raise `DoesNotExist` so it will never reach `if not product_pk:` condition. `.filter(name="Gift...").first()` would be a better option

Reply:
Yeah, this code is not great whatsoever and likely requires more work (including, for instance, how attribute values are assigned), my aim was to do minimal changes to make it work with the new products.

Reply:
Fixed.